docs/platforms/python/tracing/span-metrics/index.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Sending Span Metrics
+description: "Learn how to add attributes to spans in Sentry to monitor performance and debug applications "
+sidebar_order: 10
+---
+
+<Alert>
+
+To use span metrics, you must first <PlatformLink to="/tracing/">configure tracing</PlatformLink> in your application.
+
+</Alert>
+
+Span metrics allow you to extend the default metrics that are collected by tracing and track custom performance data and debugging information within your application's traces. There are two main approaches to instrumenting metrics:
+
+1. [Adding metrics to existing spans](#adding-metrics-to-existing-spans)
+2. [Creating dedicated spans with custom metrics](#creating-dedicated-metric-spans)
+
+## Adding Metrics to Existing Spans
+
+You can enhance existing spans with custom metrics by adding attributes. This is useful when you want to augment automatic instrumentation or add contextual data to spans you've already created.
+
+```python
+span = sentry_sdk.get_current_span()
+if span:
+    # Add individual metrics
+    span.set_attribute("database.rows_affected", 42)
+    span.set_attribute("cache.hit_rate", 0.85)
+    
+    # Add multiple metrics at once
+    span.set_attributes({
+        "memory.heap_used": 1024000,
+        "queue.length": 15,
+        "processing.duration_ms": 127
+    })
+```
+
+### Best Practices for Span Attributes
+
+When adding metrics as span attributes:
+
+- Use consistent naming conventions (for example, `category.metric_name`)
+- Keep attribute names concise but descriptive
+- Use appropriate data types (string, number, boolean, or (non-mixed) arrays of these types)
```

Comment:
The double-nested parenthesis is a bit weird, would try to rewrite in a way that does not use this. For example:

```suggestion
- Use appropriate data types (string, number, boolean, or an array containing only one of these types)
```